drvregistry/drvregistry.go

```diff
@@ -164,10 +164,10 @@ func (r *DrvRegistry) RegisterDriver(ntype string, driver driverapi.Driver, capa
 	}
 
 	r.Lock()
-	_, ok := r.drivers[ntype]
+	dd, ok := r.drivers[ntype]
 	r.Unlock()
 
-	if ok {
+	if ok && dd.driver.IsBuiltIn() {
```

Comment:
@anusha-ragunathan  Yes. If we dont do it, it wont work at all for Plugin-V2. Since V2 will deprecate V1, it is easier to document this user configuration error than to support the supported plugin-v2. WDYT ?